modules/aws-outposts-environment-info-aws.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-aws-outposts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="aws-outposts-environment-info-aws_{context}"]
+= Obtaining information from your AWS account
+
+You can use the AWS CLI (`aws`) to obtain information from your AWS account.
+
+[TIP]
+====
+You might find it convenient to store some or all of these values as environment variables by using the `export` command.
+====
+
+.Prerequisites
+
+//What do we say about this? It's up and running to the degree that you can grab required details.
+* You have an AWS Outposts site with the required hardware setup complete.
+//Not sure how to talk about this, check AWS docs.
+* You have access to your AWS account by using the AWS CLI (`aws`) as an administrator.
+
+.Procedure
+
+. List the Outposts that are connected to your AWS account by running the following command:
++
+[source,terminal]
+----
+$ aws outposts list-outposts
+----
+
+. Retain the following values from the output of the `aws outposts list-outposts` command:
+
+** The Outpost ID.
+
+** The Amazon Resource Name (ARN) for the Outpost.
+
+** The AWS Outpost availability zone ID.
++
+[NOTE]
+====
+The output of the `aws outposts list-outposts` command includes two values related to the availability zone: `AvailabilityZone` and `AvailabilityZoneId`. The value you use to configure a compute machine set that creates compute machines in AWS Outposts is `AvailabilityZoneId`.
+====
+// This is a guess on my part. The example in AWS docs has:
+// "AvailabilityZone": "us-west-2a",
+// "AvailabilityZoneId": "usw2-az1",
+// The first one looks more like a normal AZ, so I am guessing the latter is the Outposts one.
```

Comment:
The zone and zoneid here represent the same zone, we need zone (not id) for the machine set spec

The selection to be in outposts depends on us selecting the correct subnet id, but the AZ may look the same as a non-outposts worker